components/teams-service/constants/constants.go

```diff
@@ -0,0 +1,5 @@
+package constants
```

Comment:
Reduce use of literal "(unassigned)" to just one instance in teams-service, just like in other services.